src/integrated/soil_canopy_model.jl

```diff
@@ -395,18 +394,18 @@ function PlantHydraulics.root_flux_per_ground_area!(
     model::Canopy.PlantHydraulics.PlantHydraulicsModel{FT},
     Y::ClimaCore.Fields.FieldVector,
     p::NamedTuple,
-    t::FT,
-) where {FT}
+    t,
+) where {(FT)}
```

Comment:
parentheses?

Reply:
do you think it is good practice to not use the `where {FT}`/include parameteric types if we arent dispatching off them/dont need the FT in the function?

not to change in this PR. more a question for the future

Reply:
I'm not sure - I don't think we get any quantitative benefit from specifying `where {FT}` in these cases, but maybe it's a bit more readable/clear to the user? I'm fine with either way

Reply:
ok! maybe this is something we can clean up as we merge PRs that affect these functions if we decide to do this. I can see both sides :) but might lean towards removing (might be confusing to newer users of Julia?).